pkgs/build-support/setup-hooks/wrap-gapps-hook/tests/sample-project/Makefile

```diff
@@ -0,0 +1,23 @@
+PREFIX = $(out)
```

Comment:
Other tests are in `pkgs/test`, BTW, though I am sympathetic to putting tests near what is being tested, too.

Reply:
Oh, did not notice that. It might be cleaner to move it there.

Reply:
Python tests are in `pkgs/development/interpreters/python/`

Reply:
Thinking about it more, I prefer having it near as well.